x-pack/plugins/apm/public/components/routing/templates/settings_template.stories.tsx

```diff
@@ -25,6 +25,14 @@ const coreMock = {
   },
 } as unknown as Partial<CoreStart>;
 
+const configMock = {
+  featureFlags: {
+    agentConfigurationAvailable: true,
+    schemaAvailable: true,
+    indicesAvailable: true,
```

Comment:
why is this config `indicesAvailable` missing in `serverless.oblt.yml` file ?

Reply:
It's a type, should be `configurableIndicesAvailable`, I'll fix it